app/channels/concurrent_editing_channel.rb

```diff
@@ -3,7 +3,7 @@ class ConcurrentEditingChannel < ApplicationCable::Channel
 
   def subscribed
     # Called first to connect user to the channel.
-    stream_from "concurrent_editing_channel:#{params[:mapSlug]}"
+    stream_from("concurrent_editing_channel:#{params[:mapSlug]}")
```

Comment:
Were these changes from Rubocop telling you how it should be formatted? I think that's outside the scope of this PR ideally... it's fine that it's included but I'm sorry I should have caught this and made it clear you don't have to fix everyone else's syntax! Thank you though!!!

Reply:
Yep, the reason for those changes is that while I was setting up the Rubocop job I ran into lots of errors as you can see by the commit history... In my attempts to make this work, I changed the setup of the linter a bit (like adding the style guide as a gem instead of requiring it as a static file). After I made those changes, I had to run Rubocop again so it wouldn't fail due to linter offenses.